### ISSUE TITLE
Failure to use sphinx_panels?

### ISSUE THREAD

**Issue by PaulWessel** [GenericMappingTools/gmt#4923]
Doing a fresh doc build and got this:

```
Extension error:
Could not import extension sphinx_panels (exception: No module named 'sphinx_panels')
ninja: build stopped: subcommand failed.
```

Is there a revised prerequisite I need to install?  Pinging @seisman and @meghanrjones for help.

**Comment by seisman** [comment]
Yes, you need to install it.

pip install sphinx-panels

**Comment by PaulWessel** [comment]
And not via port which installed sphinx in the first place?

**Comment by seisman** [comment]
I don’t think port provides this package

**Comment by seisman** [comment]
I can add the sphinx extension to the gmt repository, so you don't have to install it yourself.

The size of the extension is about 344 KB.

**Comment by PaulWessel** [comment]
That might be better since not sure if what pip installed will be visible to the sphinx running via port.

**Comment by seisman** [comment]
The issue will be addressed in #4924. 

However, you will still see some **warnings** when building the **manpages**, because sphinx-panels is incompatible with manpages.
